src/nft/components/collection/ActivityCells.tsx

```diff
@@ -66,7 +76,7 @@ const formatListingStatus = (status: OrderStatus): string => {
     case OrderStatus.Expired:
       return 'Expired'
     case OrderStatus.Valid:
-      return 'Add to Bag'
+      return 'Unavailable'
```

Comment:
should this check isPurchaseableOrder before marking as unavailable?

Reply:
it actually already checks it before this function is called. Considering this isn't going to be used outside of this component I don't believe that the extra check would be too necessary